beamer/tests/contracts/test_fill_manager.py

```diff
@@ -2,28 +2,32 @@
 from eth_abi.packed import encode_abi_packed
 from eth_utils import keccak
 
-from beamer.tests.util import alloc_accounts, create_request_hash
+from beamer.tests.util import alloc_accounts, alloc_whitelisted_accounts, create_request_hash
 
 
-def test_fill_request(fill_manager, token, deployer):
+def test_fill_request(fill_manager, token):
     chain_id = brownie.web3.eth.chain_id
     amount = 100
-    filler, receiver = alloc_accounts(2)
+    (receiver,) = alloc_accounts(1)
+    (filler,) = alloc_whitelisted_accounts(1, {fill_manager})
 
     with brownie.reverts("Ownable: caller is not the owner"):
-        fill_manager.addAllowedLP(deployer, {"from": filler})
+        fill_manager.addAllowedLp(filler, {"from": receiver})
 
-    whitelist_tx = fill_manager.addAllowedLP(deployer, {"from": deployer})
-    assert "LPAdded" in whitelist_tx.events
+    whitelist_tx = fill_manager.addAllowedLp(receiver)
+    assert "LpAdded" in whitelist_tx.events
+    blacklist_tx = fill_manager.removeAllowedLp(receiver)
+    assert "LpRemoved" in blacklist_tx.events
```

Comment:
It's boring to read tests that do a little bit of everything, but this PR does not change it.

Reply:
Yes, I'm also not happy about that we have some tests which do a little bit of everything. That's also the case on lots of other places, but I totally agree. 

The lines you mentioned add and remove the `receiver` though not the filler, in order to test that events are created. Still misplaced.

Reply:
We should definitely refactor this